src/Microsoft.DocAsCode.Metadata.ManagedReference.Common/ExtractMetadata/Project/AbstractDocument.cs

```diff
@@ -0,0 +1,8 @@
+﻿
```

Comment:
Please add copyright header, as similar to https://github.com/surban/docfx/blob/f7bfa0a9a1b305e6a05f12ce156ae78d21f319c7/src/Microsoft.DocAsCode.Metadata.ManagedReference.Common/ExtractMetadata/InputParameters/SourceFileInputParameters.cs#L1